pkg/redirectpolicy/manager.go

```diff
@@ -499,6 +509,16 @@ func (rpm *Manager) upsertService(config *LRPConfig, frontendMapping *feMapping)
 	}
 }
 
+func (rpm *Manager) checkPodReady(pod *slimcorev1.Pod) bool {
+	ready := slimcorev1.ConditionUnknown
+	for _, cond := pod.Status.Conditions {
```

Comment:
Compilation fails on this line with
```
../pkg/redirectpolicy/manager.go:514:39: syntax error: _, cond := pod.Status.Conditions used as value
```
I think you need:
```suggestion
	for _, cond := range pod.Status.Conditions {
```

Reply:
oops